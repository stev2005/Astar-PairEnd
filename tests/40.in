CGTTTGCTACTGTATTGCTTGCGGGATCGGGTACGATATACGGCGCACGTAACTGTTGAGAGCCAACTAATCTCGCGGGCGTCTGAGCGGTGACTACAACCTGACCCTTAACTGACGGGAAGGCTGCAGCCTCTTGACCACGCAAGCGATGCCAGAGTACCCAGCAACCCCGTTGTCTCCGTGAAAAGACTCGATTATC
15
AGTACCTGACAGAGGATTTTAGAGG
GGGTTGCGGTTGCTTCGTAAATTTGTCGGACAGTTGCAAATAGCTCGCAGGACCTTCCCTCGTTCGCTGGTGGCTCGCTGACTCGGCTTGGACCAGTCCGTAACCTTTAGGATCGCTTCTCTTGCTCACGGATGGACCAGTGGGTATGTACCACTCAACCT
GGGAGACGGAAAGTCCGAGTCGACTCCAGAAAGTACTCTCCACTTGCCTTCACCCAGTACTCCGA
TTTGACGCAGACTTAATCGCCAAGCTTTC
CAGATGCCTGTTTTGATGCCTTGTCTGAGAT
ACATTCCGTAGTTATTGAACTGAAGTAATTATAATTCAGCAAAAATTTTTATGCTAAAATTAGAACTGCCTGCTGGGCCCACAGGAGGATCTTATACTGTACCGATAGACTAGATCACTAACTACTAGGGCGT
AT
ACCCA
AACCAGAATCATGTTCCCGTACATGAAAGAGGCTTCCTGACGTTCGAGTTCTACGGGGGATAGATCCCATCCCCCGTCATAGGATCTCTCCGGTGCAAGCT
GCCTTACATCGGGCTCTGGACAATGATACTAAACTTGATCGCTATGCGCTGGAGCGTCGAATAAAATGATAGATGTGTCTTACTGGCCTTGTGGTTGTCTGCCGCAGTTTGGTTCGCGTCGCAAAATGTCCCGTTTTCGGCACAT
GCCGGTTG
TGTGCATGGGGGCTAAACCC
TTCACTTCGTAAGGGGGAAAGGGGTGGATTAGTGTACGCTCGATAGCGGGTCACTTTCTGCAAAGTATGGTTATTACATTGGATTTGGCGAGGAGAAGTGAGCACACGCAGTGGGCGCTTTTCCTTCTCACGTTGATTTATCACTGGCT
AGACTAAATGCCAAAGATGTACAATAGCCGTGATTATAATGCAGCAAGTGCTAGTTGGATATCATCAGCCCTGCCACCGCATAACATTGTTGTAGGCTAGAGCGTGGCAAGAAGACGTCAGAGCACTGAAAAGGGCACGACAACTATCTAGGAATACTGCTTCCCAGCCCG
CGAATTGGTAATAATGTGTGCCTGCCTGGCAAA
