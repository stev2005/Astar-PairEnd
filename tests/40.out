11
73
29
12
13
66
0
0
47
68
2
7
74
82
14
